src/SignalR.Orleans/SignalR.Orleans.csproj

```diff
@@ -24,10 +24,10 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.SignalR.Core" Version="1.0.0-alpha2-final" />
-    <PackageReference Include="Microsoft.Orleans.Core" Version="2.0.0-*" />
-    <PackageReference Include="Microsoft.Orleans.OrleansProviders" Version="2.0.0-*" />
-    <PackageReference Include="Microsoft.Orleans.OrleansRuntime" Version="2.0.0-*" />
-    <PackageReference Include="Microsoft.Orleans.OrleansCodeGenerator.Build" Version="2.0.0-*" />
+    <PackageReference Include="Microsoft.Orleans.Core" Version="2.0.0-beta*" />

```

Comment:
fix: if you have `NuGet.config` or `nuget package source` set with Orleans Pre release repo it will download the `preview4` instead of `beta1`